pimd/pim_upstream.c

```diff
@@ -705,14 +705,14 @@ void pim_upstream_switch(struct pim_instance *pim, struct pim_upstream *up,
 
 	if (pim_addr_is_any(up->upstream_addr)) {
 		if (PIM_DEBUG_PIM_EVENTS)
-			zlog_debug("%s: RPF not configured for %s", __func__,
+			zlog_debug("%s: RP not configured for %s", __func__,
```

Comment:
I believe RPF is correct.  The up pointer can be a `S,G` or a `*,G`.  In which case it's always the RPF that we want to use.

Reply:
Hi, I suppose this is a typo.
1. upstream_addr is not the address of nexthop,  is the address of S or RP. So it should be 'RP not configured'.
2. there are other similar logs, like in function 'pim_rpf_update'.